Assignment1/ViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ViewController.swift
+//  Prob
+//
+//  Created by Dar Dar on 28.09.2023.
+//
+
+import UIKit
+
+
+class ViewController: UIViewController {
+
+    @IBOutlet var tableView: UITableView!
+    @IBOutlet weak var navbar: UINavigationBar!
+    
+    var users = [User]()
+    
+    fileprivate func handleNavBar() {
+        navbar.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            navbar.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            navbar.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            navbar.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            navbar.bottomAnchor.constraint(equalTo: tableView.topAnchor)
+        ])
+    }
+    
+    fileprivate func handleTableView() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        tableView.contentInset = UIEdgeInsets(
+            top: 45,left: 0,bottom: 0,right: 0);
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: 60).isActive = true
+    }
```

Comment:
this above two methods is ok and it is used by programmers, but instead of this I recommend you to use xib for this screen and to set all constraints there.

Reply:
Got it! Can I leave it like this? and from now on I will set the constraints in the xib file

Reply:
It seems that your constraints are not set up correctly (please refer to the image below). The leading constraint of the UITableView should not be attached to the navigation bar, and the top constraint should be below the navigation bar. Additionally, there seems to be a duplicated bottom constraint. These issues could potentially cause problems for upcoming tasks.

<img width="1158" alt="Screenshot 2023-11-06 at 19 43 49" src="https://github.com/darsius/iOS-Internship/assets/56957259/b47c2982-a37d-4f51-946f-5be656f7f892">